suite-native/transactions/src/components/TransactionDetail/TransactionDetailSummary.tsx

```diff
@@ -50,32 +51,51 @@ const VerticalSeparator = () => {
     return <Box style={applyStyle(borderLineStyle)} />;
 };
 
-const TransactionDetailSummaryRow = ({ value, title }: { value: string; title: string }) => (
-    <Box flexDirection="row" alignItems="center">
-        <TransactionDetailSummaryStepper />
-        <Box marginLeft="medium">
+export const RowWithTitle = ({ title, value }: { title: string; value: string }) => {
+    const handleCopy = () => Clipboard.setStringAsync(value);
+    return (
+        <Box>
             <Text color="gray600" variant="hint">
                 {title}
             </Text>
-            <Text numberOfLines={1} ellipsizeMode="middle" style={{ maxWidth: 160 }}>
-                {value}
-            </Text>
+            <Box flexDirection="row" justifyContent="space-between" alignItems="center">
+                <Text numberOfLines={1} ellipsizeMode="middle" style={{ width: 160 }}>
+                    {value}
+                </Text>
+                <IconButton iconName="copy" onPress={handleCopy} isRounded colorScheme="gray" />
+            </Box>
+        </Box>
+    );
+};
+
+const SummaryRow = ({
+    children,
+    leftComponent,
+}: {
+    children: React.ReactNode;
+    leftComponent: React.ReactNode;
+}) => (
+    <Box flexDirection="row" alignItems="center">
+        <Box style={{ width: 40, alignItems: 'center' }}>{leftComponent}</Box>
+        <Box marginLeft="medium" flex={1}>
+            {children}
         </Box>
     </Box>
 );
 
 const confirmationIconStyle = prepareNativeStyle<{ isTransactionPending: boolean }>(
     (utils, { isTransactionPending }) => ({
-        marginLeft: -12,
         marginRight: utils.spacings.small,
         backgroundColor: isTransactionPending ? utils.colors.yellow : utils.colors.forest,
         borderRadius: utils.borders.radii.round,
         padding: utils.spacings.small,
+        marginVertical: utils.spacings.small,
+        marginLeft: utils.spacings.small,
```

Comment:
Both `marginLeft` and `marginRight` shouldn't be necessary if it's styled right

Reply:
https://github.com/trezor/trezor-suite/pull/7272/commits/39ce90eab3ceee16f586a865709b88eb24350166